tests/baselines/reference/tsConfig/Default initialized TSConfig/tsconfig.json

```diff
@@ -40,6 +40,7 @@
     // "noImplicitReturns": true,             /* Report error when not all code paths in function return a value. */
     // "noFallthroughCasesInSwitch": true,    /* Report errors for fallthrough cases in switch statement. */
     // "noUncheckedIndexedAccess": true,      /* Include 'undefined' in index signature results */
+    // "noDeprecated": true,                  /* Issue a error if declaration is deprecated */
```

Comment:
For consistency with the other comments, consider something like "Report error when referencing deprecated declaration".